src/main/resources/logback.groovy

```diff
@@ -28,6 +28,7 @@ appender("File-Appender", FileAppender) {
 logger("org.springframework", WARN)
 logger("liquibase.executor", WARN)
 logger("io.swagger.models.parameters.AbstractSerializableParameter", ERROR)
+// logger("log4j.category.org.springframework.jdbc.core", DEBUG)
```

Comment:
A comment above this about what it does would be useful. I think I've used this before as well for logging/debugging SQL

Reply:
It didn't work. I removed the line.